### PR TITLE
V3.0.0

### PR DESCRIPTION
**Pull request by adiletelf**

### CODE REVIEW

File: src/enums/ganttRole.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    Legend = "Legend",
+    Task = "Task",
+    Parent = "Parent",
+    StartDate = "StartDate",
+    EndDate = "EndDate",
+    Duration = "Duration",
+    Completion = "Completion",
+    Resource = "Resource",
+    Milestones = "Milestones",
+    // Renaming to "Tooltips" in capabilities.json breaks the report after updating to the newer version
```
Comment (MulyukovAidar) [inline src/enums/ganttRole.ts:11]: What is this?

File: src/enums/labelForDate.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum LabelForDate {
```
Comment (MulyukovAidar) [inline src/enums/labelForDate.ts:1]: store all enums in 1 file

**Comment by MulyukovAidar** [comment]
test for highlights working properly?

File: src/enums/ganttRole.ts (hunk 1 above)
Comment (adiletelf) [reply src/enums/ganttRole.ts:11]: Forgot to delete a comment. Essentially, I tried to rename "GanttRole.ExtraInformation" to "GanttRole.Tooltips" to match a display name in capabilities.json, but then found that it's a breaking change. When replacing old version with newer version data field "Tooltips" could not be matched and will be reset.

I deleted the comment.

File: src/enums/labelForDate.ts (hunk 2 above)
Comment (adiletelf) [reply src/enums/labelForDate.ts]: fixed in 674599a

**Pull request merged**